g11n-ws/modules/md-service-authen/src/main/java/com/vmware/vip/core/csp/service/CSPTokenService.java

```diff
@@ -130,11 +130,19 @@ public void verify(JWTClaimsSet claimsSet, SecurityContext c) throws BadJWTExcep
         claim.setSub(claimSet.getClaim("sub").toString());
         claim.setExp(claimSet.getClaim("exp").toString());
         claim.setIat(claimSet.getClaim("iat").toString());
-        claim.setAcct(claimSet.getClaim("acct").toString());
-        claim.setDomain(claimSet.getClaim("domain").toString());
-        claim.setContext(claimSet.getClaim("context").toString());
         claim.setContextName(claimSet.getClaim("context_name").toString());
-        claim.setPerms(claimSet.getStringArrayClaim("perms"));
+        if (claimSet.getClaim("acct") != null){
```

Comment:
The bug's requirment is remove 'acct' and 'domain', why not remove?

Reply:
I have add the comments in issue. It needn't remove just check the null case.

Reply:
ok, please add 'It needn't remove just check the null case.' to your comment on issue to clarify change needed.